x-pack/plugins/ml/public/jobs/jobs_list/components/job_actions/results.js

```diff
@@ -14,29 +14,27 @@ import {
 } from '@elastic/eui';
 
 import chrome from 'ui/chrome';
+import moment from 'moment';
+const TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
 import { mlJobService } from 'plugins/ml/services/job_service';
 
 function getLink(location, jobs) {
   let from = 0;
   let to = 0;
   if (jobs.length === 1) {
-    from = jobs[0].earliestTimeStamp.string;
-    to = jobs[0].latestTimeStamp.string;
+    from = jobs[0].earliestTimeStampMs;
```

Comment:
Nit, but 'timestamp' is generally one word, so I think these should be `earliestTimeSsampMs` and `latestTimestampMs`!